modules/ssh/ssh.go

```diff
@@ -142,10 +143,14 @@ func sessionHandler(session ssh.Session) {
 	// Wait for the command to exit and log any errors we get
 	err = cmd.Wait()
 	if err != nil {
-		log.Error("SSH: Wait: %v", err)
+		// Cannot use errors.Is here because ExitError doesn't implement Is
+		// Thus errors.Is will do equality test NOT type comparison
+		if _, ok := err.(*exec.ExitError); !ok {
+			log.Error("SSH: Wait: %v", err)
+		}
 	}
 
-	if err := session.Exit(getExitStatusFromError(err)); err != nil {
+	if err := session.Exit(getExitStatusFromError(err)); err != nil && !errors.Is(err, io.EOF) {
```

Comment:
Is using error.Is correct here? https://github.com/golang/go/issues/39155 suggests that io.EOF never should be wrapped(and currently standard library shouldn't do that)

Reply:
Just because io.EOF shouldn't be wrapped does not mean that it cannot be wrapped and errors.Is will still work on it.

We don't care about io.EOF. we don't care about errors that accidentally wrap io.EOF.

Therefore we should ignore them.

Therefore errors.Is is correct